config/default.yml

```diff
@@ -3045,6 +3045,7 @@ Style/AccessModifierDeclarations:
     - inline
     - group
   AllowModifiersOnSymbols: true
+  AllowModifiersOnAttrs: false
```

Comment:
I wondered what the default value should be, but decided to set it to `false` for compatibility.

Reply:
Set it to `true` won't cause any existing code to suddenly have a violation, because this is not really a new rule but just an exception to an existing rule. So I think it can default to `true` to align with `AllowModifiersOnSymbols`.

Reply:
Indeed. Updated.